.github/workflows/pytest_and_ruff.yml

```diff
@@ -108,7 +96,7 @@ jobs:
         python3.10 -m pip install pytest
     # If you do not install Weave's dependencies then it will fail PyLint due to import-errors
     - name: Run pylint
-      run: pylint weave weave/tests
```

Comment:
Why did you change pylint to only test that specific test? It should run on all tests

Reply:
Great catch... I did that to speed up my testing iteration. That would've been bad to keep in. Thank you!